docker/Dockerfile.base_e2e

```diff
@@ -40,7 +40,7 @@ RUN apt-get install -y openjdk-11-jdk \
 # Install python
 RUN add-apt-repository -y ppa:deadsnakes/ppa \
     && apt-get update \
-    && apt-get install -y python3.7 python3.8 python3.9 python3.10 python3-pip python3-venv python3.8-venv python3.7-venv python3.9-venv python3.10-venv \
+    && apt-get install -y python3.7 python3.7-dev python3.8 python3.9 python3.9-dev python3.10 python3.10-dev python3-pip python3-venv python3.8-venv python3.7-venv python3.9-venv python3.10-venv \
```

Comment:
add `python3.8-dev`?

Reply:
> add `python3.8-dev`?

`python3.8-dev` is installed by ubuntu20.04 by default. But in case we upgrade the base image, i'lll add it.